docs/tutorials/notes-mvvm/index.yml

```diff
@@ -0,0 +1,56 @@
+### YamlMime:Tutorial
+title: "Upgrade your app with MVVM concepts"
+metadata:
+  title: "Tutorial: Use the MVVM toolkit to update the Notes app"
+  description: Upgrade the Notes app from the previous tutorial with the MVVM Community Toolkit.
+  audience: Developer
+  level: Beginner
+  ms.topic: interactive-tutorial
+  ms.date: 12/21/2022
+  author: adegeo
+  ms.author: adegeo
+items:
+- durationInMinutes: 1
+  content: |
+    [!INCLUDE [](includes/intro.md)]
+
+- title: Upgrade the app from .NET 6 to .NET 7
+  durationInMinutes: 3
+  content: |
+    [!INCLUDE [](includes/upgrade.md)]
+
+- title: Update the model
+  durationInMinutes: 7
+  content: |
+    [!INCLUDE [](includes/model.md)]
+
+- title: About viewmodel and About view
```

Comment:
I think this title should be changed so that it's an instruction (e.g. Upgrade the app, Update the model, and all the titles from the first tutorial). Maybe something like "Refactor the About page", or "Create the About view model", or similar.